GenderPayGap.WebUI/Views/Register/RequestReceived.cshtml

```diff
@@ -12,11 +12,11 @@
                     <h1 class="bold-medium">
                         We’ve got your details.
                         <br/>
-                        We will review them and get in touch to confirm.
+                        We will review them and email you to confirm.

```

Comment:
There is no need for conditional logic on this page because this copy change is a general improvement. (And private organisations wouldn't reach this page anyway)